modules/ztp-deploying-siteconfig-hub-side-template.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-deploying-siteconfig-hub-side-template_{context}"]
+= Deploying a SiteConfig with hub-side templating
```

Comment:
This could be interpreted that the SiteConfig is being templated, which is not. Maybe a rephrase would help here, something like "Deploying SiteConfig to support hub templated PolicyGenTemplates" or something along these lines.

Reply:
good point. I'd propose: 
"Deploying a managed cluster with `SiteConfig` and `PolicyGenTemplates` using hub-side templates"
WDYT?

Reply:
"Deploying a managed cluster with SiteConfig where PolicyGenTemplates is using hub-side templates"?